.github/workflows/test_and_deploy.yml

```diff
@@ -84,31 +87,31 @@ jobs:
         if: matrix.tool == 'pip'
         uses: aganders3/headless-gui@v2
         with:
-          run: python -m tox -vv
+          run: python -m tox
         env:
           PYVISTA_OFF_SCREEN: True  # required for opengl on windows
           NAPARI: ${{ matrix.napari }}
           FORCE_COLOR: 1
-          # PySide6 only functional with Python 3.10+
-          TOX_SKIP_ENV: ".*py39-PySide6.*"
+          TOX_SKIP_ENV: |
+            ${{ matrix.platform == 'macos-latest' && '.*PySide2.*' || '.*py3(11|12|13)-PySide2.*'  }}
 
       - name: Test with tox - conda
         if: matrix.tool == 'conda'
         uses: aganders3/headless-gui@v2
         with:
           shell: bash -el {0}
-          run: python -m tox -vv
+          run: python -m tox
         env:
           PYVISTA_OFF_SCREEN: True  # required for opengl on windows
           NAPARI: ${{ matrix.napari }}
           FORCE_COLOR: 1
-          # PySide6 only functional with Python 3.10+
-          TOX_SKIP_ENV: ".*py39-PySide6.*"
+          TOX_SKIP_ENV:  ${{ matrix.platform == 'macos-latest' && '.*py3(10|13)-PySide2.*' || '.*py3(11|12|13)-PySide2.*'  }}
```

Comment:
I don't think I get this condition, it looks like we just want to skip `PySide2` for all python versions on MacOS right? Why the `||` condition?

Reply:
Good that I have time to take a look on it after sleeping (changed `'.*py3(11|12|13)-PySide2.*' to `'.*py313-PySide2.*')

It is If on the level of action definition. For macos-latest runner, it evaluates to `.*py3(10|13)-PySide2.*` https://github.com/napari/napari-plugin-manager/actions/runs/13467739960/job/37636965822?pr=133#step:10:20

for all other runners it evaluates to `.*py3(11|12|13)-PySide2.*` (in last commit it will be `'.*py313-PySide2.*')
https://github.com/napari/napari-plugin-manager/actions/runs/13467739960/job/37636964801?pr=133#step:10:20

It is because there are no conda binaries for pyside2 and python 3.13, and it fails to install pyside2 in macOS latest.

Reply:
@DragaDoncila Told me if explanation is enough and if I should add some commnet to code?

Reply:
ah no, the explanation makes sense, maybe just adding brackets around the `&&` conditions would make it a little clearer, even though they're slightly redundant